src/config/statsd.ts

```diff
@@ -35,6 +38,22 @@ function hrtimer() {
   };
 }
 
+export const elapsedTimeMetrics = (path: string) => {
```

Comment:
Nice!

Reply:
Do you even need the path? you can get that from the request